convert-image-format@el-amine-404/files/convert-image-format@el-amine-404/convert-image-format.sh

```diff
@@ -68,10 +61,49 @@ fi
 # FALSE "pcx" "PC Paintbrush Bitmap Image" \
 # FALSE "psd" "Photoshop Document" \
 
+# full path to the selected file
+FILE="$1"
 # directory where the generated image will be saved
 DIRECTORY=$(dirname "$1")
 
 convert_image() {
+
+  if ! EXTENSION=$(
+    /usr/bin/zenity --list --radiolist \
+      --title="$IMAGE__TITLE" \
+      --text="$IMAGE__PROMPT" \
+      --height=320 \
+      --width=640 \
+      --column="$IMAGE__COLUMN_1" --column="$IMAGE__COLUMN_2" --column="$IMAGE__COLUMN_3" \
+      FALSE "apng" "Animated Portable Network Graphics" \
+      FALSE "avif" "AV1 Image File Format" \
+      FALSE "bmp" "Bitmap" \
+      FALSE "cur" "Microsoft Icon" \
+      FALSE "gif" "Graphics Interchange Format" \
+      FALSE "heic" "High Efficiency Image Coding" \
+      FALSE "heif" "High Efficiency Image Format" \
+      FALSE "ico" "Microsoft Icon" \
+      FALSE "j2k" "JPEG 2000 Code Stream" \
+      FALSE "jfi" "JPEG (Joint Photographic Experts Group)" \
+      FALSE "jfif" "JPEG (Joint Photographic Experts Group)" \
+      FALSE "jif" "JPEG (Joint Photographic Experts Group)" \
+      FALSE "jp2" "JPEG 2000 Image" \
+      FALSE "jpe" "JPEG (Joint Photographic Experts Group)" \
+      FALSE "jpeg" "JPEG (Joint Photographic Experts Group)" \
+      FALSE "jpg" "JPEG (Joint Photographic Experts Group)" \
+      FALSE "pjp" "JPEG (Joint Photographic Experts Group)" \
+      FALSE "pjpeg" "JPEG (Joint Photographic Experts Group)" \
+      FALSE "pdf" "Portable Document Format" \
+      FALSE "png" "Portable Network Graphics" \
+      FALSE "svg" "plaintext Scalable Vector Graphics" \
+      FALSE "svgz" "compressed Scalable Vector Graphics" \
+      FALSE "tif" "Tagged Image File Format" \
+      FALSE "tiff" "Tagged Image File Format" \
+      FALSE "webp" "Web Picture format"
```

Comment:
I've gone ahead and updated the translation files for you but you may want to consider if these dialogs are worth translating as well. To update the translations locally, you can run `./cinnamon-spices-makepot convert-image-format@el-amine-404` from the root of the repository.